### DISCUSSION TITLE
not possible to obtain Graph using docker container (0.6)

### DISCUSSION THREAD

**Discussion by cherkavi** [JanusGraph/janusgraph-docker#133] [Q&A]
1. Start docker container
```sh
DOCKER_TAG=0.6
docker run --rm --name janusgraph-default --volume `pwd`:/workspace:rw --network="host" janusgraph/janusgraph:$DOCKER_TAG
```

2. Connect with Gremlin
```sh
docker exec -e GREMLIN_REMOTE_HOSTS=localhost -it  janusgraph-default ./bin/gremlin.sh
```

3. According to tutorial:
https://docs.janusgraph.org/getting-started/basic-usage/#loading-with-an-index-backend
```sh
:remote connect tinkerpop.server conf/remote.yaml

# !!! no such file in /opt/janusgraph/conf: janusgraph-berkeleyje-es.properties
graph = JanusGraphFactory.open('conf/janusgraph-berkeleyje-es.properties'
```

**Comment by FlorianHockmann** [comment]
I moved this over to Discussions as this is more a usage question than a bug report or feature request.

The problem is that this part of the docs assumes that you using the zip archive as described [under _Local Installation_  in the Getting Started docs](https://docs.janusgraph.org/master/getting-started/installation/#local-installation) instead of the Docker container. That explains why this config file isn't present at all.

But apart from that, `JanusGraphFactory.open()` doesn't make much sense if you're using the Docker image as that already includes JanusGraph Server which opens a graph on startup. It also already includes an index backend by default. Although that's Apache Lucene instead of Elasticsearch.

Unfortunately the _Basic Usage_ chapter was written at a time when JanusGraph didn't have a Docker image and we didn't update that part after making the Docker image (and JanusGraph Server in general) the preferred way to use JanusGraph. We should definitely improve this situation so I created an issue for that: [#3785](https://github.com/JanusGraph/janusgraph/issues/3785).
